espnet2/bin/asr_inference.py

```diff
@@ -309,7 +310,13 @@ def inference(
             batch = {k: v[0] for k, v in batch.items() if not k.endswith("_lengths")}
 
             # N-best list of (text, token, token_int, hyp_object)
-            results = speech2text(**batch)
+            try:
+                results = speech2text(**batch)
+            except TooShortUttError:
+                logging.info(
+                    "Utterance is too short for subsampling, return empty results"
```

Comment:
Is it possible to add the following information?
1. utterance ID (if possible)
2. how many (subsampled?) frames of the utterance ID
3. how many (subsampled?) frames are required?

e.g, "Utterance [utterance ID] has XXX frames and is too short for subsampling (it needs more than YYY frames), return empty results"

Reply:
Sure, let me have a try on it